testhelpers/docker_registry.go

```diff
@@ -11,25 +11,33 @@ import (
 	"net/http/httptest"
 	"path/filepath"
 	"strconv"
+	"strings"
 	"testing"
 
 	"github.com/google/go-containerregistry/pkg/registry"
 )
 
 type DockerRegistry struct {
-	Host            string
-	Port            string
-	Name            string
-	server          *httptest.Server
-	DockerDirectory string
-	username        string
-	password        string
-	regHandler      http.Handler
-	authnHandler    http.Handler
+	Host             string
+	Port             string
+	Name             string
+	server           *httptest.Server
+	DockerDirectory  string
+	username         string
+	password         string
+	regHandler       http.Handler
+	authnHandler     http.Handler
+	customHandler    http.Handler
```

Comment:
Do we need this? Is it set anywhere?

Reply:
Oh. I just noticed I am not using it! let me removes it

Reply:
Removed in the following [commit](https://github.com/buildpacks/imgutil/pull/125/commits/4261e2896afc176ea6a4967a4bd255f780340892)